src/DIRAC/TransformationSystem/Client/TransformationFilesStatus.py

```diff
@@ -0,0 +1,49 @@
+"""
+This module contains constants and lists for the possible Transformation files states.
+"""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+
+__RCSID__ = "$Id$"
+
+#
```

Comment:
```suggestion
#:
```
So that the constants are displayed in the documentation of the module. Same for the other lines.
(I think this can be just `#:`)